client/src/components/existing-request/existing-request-display.vue

```diff
@@ -654,4 +721,7 @@ export default class ExistingRequestDisplay extends Mixins(
     font-weight: bold;
   }
 }
+.italic {
+  font-style: italic;
```

Comment:
Use built-in Vuetify helper class instead.
See https://vuetifyjs.com/en/styles/text-and-typography/#typography

Or use one of our existing body styles if one suits you.
See https://github.com/bcgov/namerequest/blob/master/client/src/assets/scss/overrides.scss

Reply:
Used vuetify class instead, removed this css